sdk/cosmos/azure-cosmos-spark-fabric-account-data-resolver/pom.xml

```diff
@@ -0,0 +1,864 @@
+              <includeTestSourceDirectory>true</includeTestSourceDirectory>
+              <failOnWarning>true</failOnWarning>
+              <sourceDirectory>${project.basedir}/src/main/scala</sourceDirectory>
+              <testSourceDirectory>${project.basedir}/src/test/scala</testSourceDirectory>
+              <configLocation>${project.basedir}/scalastyle_config.xml</configLocation>
+              <outputFile>${project.build.outputDirectory}/scalastyle-output.xml</outputFile>
+              <outputEncoding>UTF-8</outputEncoding>
+            </configuration>
+            <executions>
+              <execution>
+                <id>validate-style</id>
+                <phase>validate</phase>
+                <goals>
+                  <goal>check</goal>
+                </goals>
+              </execution>
+              <execution>
+                <id>verify-style</id>
+                <phase>verify</phase>
+                <goals>
+                  <goal>check</goal>
+                </goals>
+              </execution>
+            </executions>
+          </plugin>
+          <!-- Shading has been moved to a separate profile since there is a bug which may result in it deadlocking during a parallel build. -->
+          <!-- See https://issues.apache.org/jira/projects/MSHADE/issues/MSHADE-384 -->
+          <!-- Once this issue is resolved this can be moved back into the build-scala profile. -->
+          <plugin>
+            <groupId>org.apache.maven.plugins</groupId>
+            <artifactId>maven-shade-plugin</artifactId>
+            <version>3.6.0</version> <!-- {x-version-update;org.apache.maven.plugins:maven-shade-plugin;external_dependency} -->
+            <executions>
+              <execution>
+                <id>02-shade</id>
+                <phase>package</phase>
+                <goals>
+                  <goal>shade</goal>
+                </goals>
+                <configuration>
+                  <relocations>
+                    <relocation>
+                      <pattern>com.azure</pattern>
+                      <shadedPattern>${shadingPrefix}.com.azure</shadedPattern>
+                      <excludes>
+                        <exclude>com.azure.cosmos.spark.*</exclude>
+                        <exclude>com.azure.cosmos.spark.fabric.*</exclude>
+                        <exclude>com.azure.cosmos.spark.diagnostics.*</exclude>
+                        <exclude>com.azure.cosmos.spark.udf.*</exclude>
+                        <exclude>com.azure.cosmos.implementation.SparkBridgeInternal</exclude>
+                        <exclude>com.azure.cosmos.models.CosmosParameterizedQuery</exclude>
+                      </excludes>
+                    </relocation>
+                    <relocation>
+                      <pattern>scala.compat.java8</pattern>
+                      <shadedPattern>${shadingPrefix}.scala.compat.java8</shadedPattern>
+                    </relocation>
+                    <relocation>
```

Comment:
I am curious - do we really need to shading these libraries? does not seem it used in the resolver, mostly they are used in spark connector, but we rely on provided spark connector from the runtime anyway